test/hotspot/jtreg/vmTestbase/vm/runtime/defmeth/AccessibilityFlagsTest.java

```diff
@@ -26,6 +26,7 @@
  *
  * @modules java.base/jdk.internal.org.objectweb.asm:+open java.base/jdk.internal.org.objectweb.asm.util:+open
```

Comment:
Can we remove all these `@modules java.base/jdk.internal.org.objectweb.asm` references?

Reply:
Not yet, as ClassFileGenerator.java is still using the package **jdk.internal.org.objectweb.asm** which is visible from these classes importing asm, they will be removed once the ClassFileGenerator.java fully uses the CP API